code/go/0chain.net/smartcontract/vestingsc/config_test.go

```diff
@@ -91,3 +94,89 @@ func TestVestingSmartContract_getConfigHandler(t *testing.T) {
 	require.NoError(t, err)
 	assert.EqualValues(t, configured, resp)
 }
+
+func TestVestingSmartContractUpdate(t *testing.T) {
+
+	var (
+		vsc            = newTestVestingSC()
+		balances       = newTestBalances()
+		tx             = newTransaction(owner, vsc.ID, 0, common.Now())
+		originalConfig = configureConfig()
+		err            error
+		currentConfig  *config
+		update         = &config{}
+	)
+
+	// 1. Malformed update
+	t.Run("malformed update", func(t *testing.T) {
```

Comment:
For consistancy can now we adopt the [table driven subtest style](https://blog.golang.org/subtests#TOC_4.) of @MurashovVen's unit tests.